crates/wash-cli/src/bin/wash.rs

```diff
@@ -220,7 +220,7 @@ enum CliCommand {
 #[tokio::main]
 async fn main() {
     use clap::CommandFactory;
-    env_logger::init();
+    tracing::init();
```

Comment:
I don't think this is a function in `tracing`

Reply:
Yep as a heads up it will be in `tracing_subscriber`